packages/scribe/tsconfig.json

```diff
@@ -1,12 +1,20 @@
 {
   "extends": "../../tsconfig.base",
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": [
```

Comment:
I like what you have done with this to clean-up import paths - good job. I've added it to my TODO list.

Reply:
Thanks.